pubsub/google/cloud/pubsub_v1/subscriber/message.py

```diff
@@ -208,8 +212,8 @@ def lease(self):
         """Inform the policy to lease this message continually.
 
         .. note::
-            This method is called by the constructor, and you should never
-            need to call it manually.
+            By default this method is called by the constructor, and you should
+            never need to call it manually.
```

Comment:
Is "never" correct here?  What if the application constructs the message with `lease=False`?

Reply:
Re-reading it again, it is not - it refers to the default case, but that's not clear from the sentence, will reword.